src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -87,6 +109,48 @@ public Title getTitle() {
         return title;
     }
 
+    /**
+     * Gets the transfer matrix. Every (i, j) entry reflects how much i receives
+     * from j. Negative amounts means i has to give j money.
+     */
+    public ArrayList<ArrayList<Double>> getTransferMatrix() {
```

Comment:
@podocarp I assume we call this method when we need to obtain the transfer matrix to show to the user?

Reply:
yes